packages/console/src/pages/Connectors/index.tsx

```diff
@@ -109,10 +166,10 @@ function Connectors() {
       }}
       pageMeta={{ titleKey: 'connectors.page_title' }}
       createButton={conditional(
-        isSocial && {
-          title: 'connectors.create',
+        isCreatable && {
+          title: createButtonTitle,
           onClick: () => {
-            navigate(buildCreatePathname(ConnectorType.Social));
+            navigate(buildCreatePathname(type[0]));
```

Comment:
📝  `type` being an array here is really ugly, didn't like it from the start but kept moving. Have a look at improving `connectorTypeTabMap` usage